API/projectClass/customFileClass.m

```diff
@@ -265,7 +272,14 @@ function displayTable(obj)
                     functionName = thisRow{3};
                     thisType = thisRow{4};
                     thisPath = thisRow{5};
+                    if isempty(thisPath)
+                        thisPath = pwd;
+                    end
                     libpath = fullfile(thisPath, thisFile);
+                    if ~exist(libpath, 'file') && ~strcmpi(thisType, supportedLanguages.Matlab.value)
+                        msg = 'The custom file (%s) cannot be found. Check that the path and filename is correct (%s).';
+                        throw(exceptions.invalidPath(sprintf(msg, thisFile, strrep(libpath, '\', '/'))));
+                    end
```

Comment:
What happens if the file does not exist and we are using matlab as the language?

Reply:
Matlab files used the Matlab search path so it will tell the user it cannot be found on the path hence I thought it irrelevant to check here

Reply:
That happens quite a bit later right? Isn't it better to flag the problem earlier?